recipes/minizip/1.2.11/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+cmake_minimum_required(VERSION 3.8)
```

Comment:
Was this Cmake file extracted from other place?

Reply:
yes, from vcpkg: https://github.com/microsoft/vcpkg/blob/master/ports/minizip/CMakeLists.txt

Reply:
Can you please add that link as a comment in the CMakeLists.txt content? Something like "Extracted from"

Reply:
good idea!

Reply:
done